packages/generate/src/cli.ts

```diff
@@ -241,6 +241,12 @@ const run = async () => {
       case "--future":
         options.future = true;
         break;
```

Comment:
@CarsonF is the idea here to provide every flag separately because I thought we also want one flag that includes all future features (I'd personally probably use just that)?

Reply:
Gotcha. That's cool with me!